x-pack/plugins/osquery/cypress/tasks/integrations.ts

```diff
@@ -145,3 +160,12 @@ export const installPackageWithVersion = (integration: string, version: string)
     method: 'POST',
   });
 };
+
+const extractSemanticVersion = (str: string) => {
```

Comment:
Cant we use https://www.npmjs.com/package/semver#clean ?

Reply:
Thanks, I think it could work! I will create another Pr to change that if that's fine with you. This Pr has been up for already way too long, and it's blocking @aleksmaus